### PR TITLE
Updated xnnpack to commit f0cb70a1ae68c4b99047b7c5d26c40fa3a65f0e4 for cmake build.

### PR DESCRIPTION
**Pull request by JerryShih**
Update for fp16 downloading path.

### CODE REVIEW

**Comment by JerryShih** [comment]
I don't know if we need to update other dependent library too.
The bazel build updating also changes the pthreadpool and cpu-info.
https://github.com/tensorflow/tensorflow/commit/8060cb22de7115f4cc3e140a057ee7412ea3bbca

**Comment by abattery** [comment]
@multiverse-tf could you review this PR?

**Review by multiverse-tf — APPROVED** [review]

File: tensorflow/lite/tools/cmake/modules/xnnpack.cmake [hunk 1]
```diff
@@ -22,7 +22,7 @@ include(FetchContent)
 OverridableFetchContent_Declare(
   xnnpack
   GIT_REPOSITORY https://github.com/google/XNNPACK
-  GIT_TAG 01c341b597504643081ff596d8ee755bf4c59c51
+  GIT_TAG f7baa13a4a78fbaf7dd1664b5eb692b47512889a
   GIT_PROGRESS TRUE
   PREFIX "${CMAKE_BINARY_DIR}"
   SOURCE_DIR "${CMAKE_BINARY_DIR}/xnnpack"
```
Comment (multiverse-tf) [inline tensorflow/lite/tools/cmake/modules/xnnpack.cmake:28]: This file just got updated in https://github.com/tensorflow/tensorflow/commit/0834c9410bb83e967973ef74a53ceeb54c12d339 w/ a more recent XNNPACK git tag. @JerryShih, could you update your branch to the latest master? Thx!

**Review by multiverse-tf — CHANGES_REQUESTED** [review]
Need to resolve the conflict.

**Comment by JerryShih** [comment]
@multiverse-tf
Do you mean use the latest XNNPACK commit or just rebase this pr?
If just rebase this pr, only the FP16 needs to be upgraded in this pr.

**Review by multiverse-tf — APPROVED** [review]

**Comment by multiverse-tf** [comment]
> @multiverse-tf
> Do you mean use the latest XNNPACK commit or just rebase this pr?
> If just rebase this pr, only the FP16 needs to be upgraded in this pr.

Just rebase. Thx for the quick response!

**Comment by JerryShih** [comment]
@multiverse-tf 
Do you close this pr accidentally or this pr is useless anymore?

**Comment by multiverse-tf** [comment]
> @multiverse-tf
> Do you close this pr accidentally or this pr is useless anymore?

Sorry, I accidentally closed the PR :-(. Let me reopen it.

**Pull request merged**